primitives/messages/src/source_chain.rs

```diff
@@ -173,6 +173,18 @@ impl OnDeliveryConfirmed for Tuple {
 	}
 }
 
+/// Handler for messages have been accepted
+pub trait OnMessageAccepted {
+	/// Called when a message has been accepted by message pallet.
+	fn on_messages_accepted(message: &MessageNonce) -> Weight;
```

Comment:
Please add `lane: &LaneId` argument here - we support multiple message lanes, so callee should know this information to be able to identify message.